src/login/logind-dbus.c

```diff
@@ -2001,7 +2001,7 @@ static int verify_shutdown_creds(
 
         if (blocked) {
                 /* We don't check polkit for root here, because you can't be more privileged than root */
-                if (uid == 0 && (flags & SD_LOGIND_ROOT_CHECK_INHIBITORS))
+                if (uid == 0)
                         return sd_bus_error_setf(error, SD_BUS_ERROR_ACCESS_DENIED,
                                                  "Access denied to root due to active block inhibitor");
```

Comment:
so this check makes no sense to me. not in the old version and not in the new either. We should always ask PK here.

This would require adding a new flag to bus_verify_polkit_async_full() which when set would bypass the internal PK bypass it does for priv clients, and force that we ask PK. Maybe turn the existing `interactive` boolean param into a flags field.

Reply:
ok, this is pre-existing, but done